examples/hello/group_vars/secret/patient0/development/richie/credentials.vault.yml

```diff
@@ -0,0 +1,24 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
Are you sur that this file is linked to the hello exemple ?

Reply:
:+1: removed it